Enhanced_Zeus_Modules.sqf

```diff
@@ -14076,6 +14089,42 @@ MAZ_EZM_fnc_initFunction = {
 			["Vehicle repaired.","addItemOk"] call MAZ_EZM_fnc_systemMessage;
 		};
 
+		HYPER_EZM_fnc_setPlateNumber = {
+			params ["_entity"];
+			if (!(_entity isKindOf "Car_F")) exitWith {
+				["This is not a vehicle!","addItemFailed"] call MAZ_EZM_fnc_systemMessage;
+			};
+			[
+				"Set License Plate Number",
+				[ 
+					[
+						"EDIT",
+						"Plate Number (12 chars max)",
+						[ 
+							"", 
+							1 
+						]
+					]
+				], 
+				{
+					params ["_values", "_args", "_display"];
+					private _entity = _args;
+					private _inputText = _values select 0;
+					if (count _inputText > 12) then {
+						_inputText = _inputText select [0, 12];
+					};
```

Comment:
The wiki says 15 is the maximum. Haven't tested

https://community.bistudio.com/wiki/setPlateNumber

Reply:
I tested it, 12 capital characters reaches the very edge of all plates, 15 runs off the side and becomes floating text

Reply:
![image](https://github.com/user-attachments/assets/c24316e0-fadc-48e6-8ea0-09dff715d233)
I typed `ABCDE` as well and it only fits up to 12 (`AB`)